### PR TITLE
Add new maintainer

### PR DESCRIPTION
**Pull request by jtpio**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Related to https://github.com/jupyterlab-contrib/jupyterlab-contrib.github.io/issues/18

### CODE REVIEW

**Comment by jtpio** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -41,3 +41,4 @@ about:
 extra:
   recipe-maintainers:
     - jtpio
+    - fcollonval
```
Comment (jtpio) [inline recipe/meta.yaml:44]: cc @fcollonval if you agree to be listed there

**Comment by jtpio** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (fcollonval) [reply recipe/meta.yaml:44]: I agree 
Thanks @jtpio

**Pull request merged**